File: library/src/main/java/com/pengrad/telegrambot/model/request/ChatAction.java
package com.pengrad.telegrambot.model.request;

import java.io.Serializable;

/**
 * stas
 * 10/21/15.
 */
public enum ChatAction implements Serializable {
    typing, upload_photo, record_video, upload_video, record_audio, upload_audio, upload_document, find_location;
    private final static long serialVersionUID = 0L;
}

File: library/src/main/java/com/pengrad/telegrambot/model/request/ChatAction.kt
package com.pengrad.telegrambot.model.request

/**
 * stas
 * 10/21/15.
 */
@Suppress("EnumEntryName")
enum class ChatAction {
    typing,
    upload_photo,
    record_video,
    upload_video,
    record_voice,
    upload_voice,
    upload_document,
    choose_sticker,
    find_location,
    record_video_note,
    upload_video_note
}
